tests/Testcontainers.Minio.Tests/MinioContainerTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.IO;
+using Amazon.S3;
+using Amazon.S3.Model;
+
+namespace TestContainers.Minio.Tests.Container;
+
+public sealed class MinioContainerTests : IAsyncLifetime
+{
+    private readonly MinioContainer _minioTestcontainer;
+
+    public MinioContainerTests()
+    {
+        _minioTestcontainer = new MinioBuilder().Build();
+    }
+
+    [Fact]
+    public async Task TestMinio()
+    {
+        const string bucketName = "somebucket";
+        await _minioTestcontainer.StartAsync();
+        var config = new AmazonS3Config
+        {
+            AuthenticationRegion = "eu-west-1",
+            ServiceURL = _minioTestcontainer.GetMinioUrl(),
+            UseHttp = true,
+            ForcePathStyle = true
+        };
+        var s3 = new AmazonS3Client(_minioTestcontainer.GetAccessId(), _minioTestcontainer.GetAccessKey(), config);
+
+        await s3.PutBucketAsync(bucketName);
+
+        var buckets = await s3.ListBucketsAsync();
+
+        Assert.NotNull(buckets);
+        Assert.NotNull(buckets.Buckets);
+        Assert.NotEmpty(buckets.Buckets);
+        Assert.Contains(buckets.Buckets, bucket => bucket.BucketName == bucketName);
+    }
+    
+    [Fact]
+    public async Task TestInsertAndGetDataFromMinio()
+    {
+        const string bucketName = "somebucket2";
+        const string fileName = "jp2137.jpg";
```

Comment:
Can we use a different file? Maybe something we generate during the test in the temp directory, like we are doing here: https://github.com/testcontainers/testcontainers-dotnet/blob/33d1a2c856d4469373d7a1c853b3d602e12a98b9/tests/Testcontainers.Tests/Unit/Containers/Unix/CopyResourceMappingContainerTest.cs#L35

Reply:
fixed